openscap_report/cli.py

```diff
@@ -18,23 +18,23 @@
                "a SCAP-compatible utility scan. Unless the --output option is specified "
                "the report will be written to the standard output.")
 LOG_LEVES_DESCRIPTION = (
-    "LOG LEVELS:\n"
-    "\tDEBUG - Detailed information, typically of interest only for diagnosing problems.\n"
-    "\tINFO - A confirmation that things are working as expected.\n"
+    "\nLOG LEVELS:\n"
```

Comment:
For these types of strings where you want to preserve whitespace etc. a triple-quoted string `""" ... """` could work better.